### PR TITLE
Upgrade to JDK 17

### PR DESCRIPTION
**Pull request by masum-islam84**
Upgrade to JDK 17

### CODE REVIEW

**Comment by masum-islam84** [comment]
Please review this and let me for any improvement suggestions.

File: .github/workflows/microservices.yml [hunk 1]
```diff
@@ -10,10 +10,11 @@ jobs:
...
         path: DCSA-TNT
         submodules: recursive
-    - name: Set up Java JDK
-      uses: actions/setup-java@v1
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
       with:
-        java-version: 11
+        java-version: 17
+        distribution: 'adopt'
```
Comment (FinnAtAsseco) [inline .github/workflows/microservices.yml:17]: adopt is no longer being updated, please use temurin instead

File: pom.xml [hunk 2]
```diff
@@ -16,13 +16,17 @@
 	<description>Demo project for Spring Boot</description>
```
Comment (FinnAtAsseco) [inline pom.xml:16]: please look at how the upgrade was done in EBL and BKG (beta2 branch). If you set dcsa-bom as parent you can remove a lot of the config from here as it then gets it from dcsa-bom

File: pom.xml [hunk 3]
```diff
@@ -36,14 +40,40 @@
...
-	</dependencies>	
+	</dependencies>
 
 	<build>
 		<plugins>
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
```
Comment (FinnAtAsseco) [inline pom.xml:51]: Not needed if you inhereit from dcsa-bom

Comment (masum-islam84) [reply pom.xml]: Done.

File: pom.xml (hunk 2 above)
Comment (masum-islam84) [reply pom.xml:17]: Done

File: .github/workflows/microservices.yml (hunk 1 above)
Comment (masum-islam84) [reply .github/workflows/microservices.yml]: done

**Review by masum-islam84 — COMMENTED** [review]
All changes was done

**Review by masum-islam84 — COMMENTED** [review]
done

**Review by FinnAtAsseco — APPROVED** [review]

**Comment by masum-islam84** [comment]
Thx for the review. Merge JDK 17 update to the master branch.

**Pull request merged**